Dockerfile

```diff
@@ -1,16 +1,11 @@
-FROM ubuntu:xenial
+FROM ubuntu:bionic
 
 # System dependencies
-RUN apt-get update && apt-get install --yes curl xz-utils
-
-# Get nodejs
-RUN mkdir /usr/lib/nodejs && \
-    curl https://nodejs.org/dist/v6.11.3/node-v6.11.3-linux-x64.tar.xz | tar -xJ -C /usr/lib/nodejs && \
-    mv /usr/lib/nodejs/node-v6.11.3-linux-x64 /usr/lib/nodejs/node-v6.11.3
-
-# Set nodejs paths
-ENV NODEJS_HOME=/usr/lib/nodejs/node-v6.11.3
-ENV PATH=$NODEJS_HOME/bin:$PATH
+RUN apt-get update && \
+    apt-get install curl xz-utils --yes && \
+    curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
+    apt-get update && \
+    apt-get install --yes nodejs
```

Comment:
Can't we just move `curl xz-utils` here as well, so we don't need to do `apt-get install` twice?

Reply:
I still need to install curl to be able to install nodejs

Reply:
Oh lol. Yes you're right =)